manifests/repo/apt.pp

```diff
@@ -77,10 +78,10 @@
             /i\d86/      => 'i386',
             default      => false,
           }
-          if versioncmp($release, '3.12') < 0 {
-            $repo_url  = "http://download.gluster.org/pub/gluster/glusterfs/${release}/LATEST/Debian/${::lsbdistcodename}/apt/"
```

Comment:
@gnif Why did you changed the variable? It does only contain the version, but the spec tests are running against a specific release.

Reply:
Because the URL is incorrect. This doesn't break the specific release the spec tests are running against as it will still select the correct version.